### ISSUE TITLE
Rapid rotations during short camera paths

### ISSUE THREAD

**Issue by WeirdRubberDuck** [OpenSpace/OpenSpace#1947]
Since the rotation interpolation is done over the traveled distance of the camera path, the rotation can be extremely fast for short camera paths (almost infinitely fast)

For these scenarios (and actually most scenarios, if possible), we instead want a solution that interpolates based on time instead of position, and that can keep interpolating even if the camera has reached the target position. That is, the camera path should not be finished until both position and rotation interpolation are finished. 

Note that this means that we need to compute how long the path takes to travel before starting the motion of the camera

**Comment by WeirdRubberDuck** [comment]
Decided to solve it for just **linear**  paths for the release, and leave the rest as future work
